router/diag_cntl.h

```diff
@@ -35,6 +35,23 @@
 #include "peripheral.h"
 #include "masks.h"
 
+struct diag_id_tbl_t {
+	struct list_head link;
+	uint8_t diag_id;
+	uint8_t pd_val;
```

Comment:
I dont see pd_val being used anywhere, I think we can remove it?

Reply:
yeah,  it has been removed